src/enums/modal.ts

```diff
@@ -31,9 +31,7 @@ export enum ModalType {
   ConfirmDeauthorization = "CONFIRM_DEAUTHORIZATION",
   DeauthorizationCompleted = "DEAUTHORIZATION_COMPLETED",
   DeauthorizationInitiated = "DEAUTHORIZATION_INITIATED",
-  Analytics = "ANALYTICS",
```

Comment:
Can we additionaly remove `analyicsReport` url from `externalHref.ts`?

https://github.com/threshold-network/bitcoin-on-base/blob/7a6d3149a786db8c6d1ad199dc22fd5c668d55c6/src/enums/externalHref.ts#L24

It's not used anywhere and it looks like it never was 😅

Reply:
Ref commit: de5c51c9